docs/kyma-release-notes/2023-10-02-release-notes-2.19.md

```diff
@@ -0,0 +1,36 @@
+
+## Application Connectivity
+
+Additional logging options has been added to Central Application Gateway.
+https://github.com/kyma-project/kyma/#debugging
+
+## Observability
+
+Pre-note: the monitoring gets removed with Kyma 2.20 as announced in this [blog post](https://blogs.sap.com/2023/09/07/removal-of-prometheus-grafana-based-monitoring-in-sap-btp-kyma-runtime/)
+
+Update of components:
```

Comment:
```suggestion
Update of the software stack:
```
My pet peeve, I hate using "component" here because that term is already used in the context of "Kyma component".